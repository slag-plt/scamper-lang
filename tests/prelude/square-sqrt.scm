(square 0.71)
(square 111)
(square 6.1)
(square 0.69)
(square 0.10000000000000009)
(square 0)
(sqrt 0.5041)
(sqrt 12321)
(sqrt 37.21)
(sqrt 0.4761)
(sqrt 0.01)
(sqrt 0)