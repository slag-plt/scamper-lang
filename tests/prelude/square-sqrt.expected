0.5041
12321
37.209999999999994
0.4760999999999999
0.010000000000000018
0
0.71
111
6.1
0.6900000000000001
0.1
0
